pulp_2_tests/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -525,3 +529,53 @@ def test_all(self):
             RPM_UNSIGNED_FEED_COUNT,
             repo['content_unit_counts']['rpm'],
         )
+
+
+class SyncZchunkRepoSkipTestCase(unittest.TestCase):
+    """Sync feed with ``zchunks`` and ensure no ``zchunk`` units exist.
```

Comment:
Unfortunately our current linters are not enforcing the line length of 79 characters. Please, assure that docstrings are following this length.